boards/feather-nrf52840/doc.txt

```diff
@@ -0,0 +1,42 @@
+/**
+@defgroup    boards_feather-nrf52840 Adafruit Feather nRF52840 Express
+@ingroup     boards
+@brief       Support for the Adafruit Feather nRF52840 Express
+
+### General information
+
+[The Feather nRF52840 Express][feather-nrf52840] is a development board
+from Adafruits Feather board family. It provides native USB support, Bluetooth
+Low Energy and IEEE 802.15.4 support via the nRF52840 MCU.
+
+![top-down view on feather-nrf52840][top-down view]
+
+[feather-nrf52840]: https://learn.adafruit.com/introducing-the-adafruit-nrf52840-feather/
+[top-down view]: https://cdn-learn.adafruit.com/assets/assets/000/068/578/medium800/circuitpython_Screenshot_2019-01-02_at_12.04.27.png
+
+### Flash the board
+
+See the **Flashing** section in @ref boards_common_nrf52. The easiest way is to
+use an external Segger J-Link Progammer connected to the [SWD Connector].
+
+[SWD Connector]: https://learn.adafruit.com/introducing-the-adafruit-nrf52840-feather/pinouts#swd-connector-3-12
+
+### Terminal
```

Comment:
Can someone more knowledgeable about this topic please read through this section. I'm not 100% sure I used the correct terms here :sweat_smile:

Reply:
LGTM